code/modules/reagents/chemistry/reagents/toxins.dm

```diff
@@ -1273,6 +1273,33 @@
 	..()
 	chosen_timer = rand(10, 30)
 
+/datum/reagent/peaceborg/confuse
```

Comment:
Why not just have these as `/datum/reagent/confuse` and `/datum/reagent/tire`?
`/datum/reagent/peaceborg` doesn't currently exist, so there's no real benefit other than having them both grouped together.